### PR TITLE
Quick updates

### PR DESCRIPTION
**Pull request by ryan1604**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-![](https://via.placeholder.com/100.png?text=Photo) |  Ryan Chua  | [Github](https://github.com/ryan1604) | [Portfolio](docs/team/ryanchua.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Neo Min Wei | [Github](https://github.com/NeoMinWei) | [Portfolio](docs/team/NeoMinWei.md)
-![](https://via.placeholder.com/100.png?text=Photo) |  Frederick  | [Github](https://github.com/) | [Portfolio](docs/team/frederick.md)
+| Display                                             |    Name     |             Github Profile              |             Portfolio             |
+|-----------------------------------------------------|:-----------:|:---------------------------------------:|:---------------------------------:|
+| ![](https://via.placeholder.com/100.png?text=Photo) |  John Doe   |      [Github](https://github.com/)      | [Portfolio](docs/team/johndoe.md) |
+| ![](https://via.placeholder.com/100.png?text=Photo) |   Don Joe   |      [Github](https://github.com/)      | [Portfolio](docs/team/johndoe.md) |
+| ![](https://via.placeholder.com/100.png?text=Photo) |  Ryan Chua  |  [Github](https://github.com/ryan1604)  |   [Portfolio](team/ryan1604.md)   |
```
Comment (NeoMinWei) [inline docs/AboutUs.md:7]: Nicely done

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+| ![](https://via.placeholder.com/100.png?text=Photo) | Neo Min Wei | [Github](https://github.com/NeoMinWei)  |  [Portfolio](team/neominwei.md)   |
+| ![](https://via.placeholder.com/100.png?text=Photo) |  Frederick  | [Github](https://github.com/wwweert123) |  [Portfolio](team/wwweert123.md)  |
```
Comment (wwweert123) [inline docs/AboutUs.md:9]: Upz

File: docs/README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
-# Duke
+# Financial Planner
```
Comment (wwweert123) [inline docs/README.md:1]: Nice!

**Review by wwweert123 — APPROVED** [review]

**Pull request merged**